### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ue71603**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,17 +32,24 @@ at [CEN](https://www.cen.eu).
...
+You can contribute by:
+* Pointing out [issues](https://github.com/VDVde/OJP/issues)
+* Creating pull requests: Use the [development documentation](https://github.com/VDVde/OJP/wiki/WG:-:-Working-with-this-repository)
+
 ### Branches
 * The master branch is currently set to OJP 1.0.1
-* Development occurs currently in the changes_for_v1.1 branch.
+* Development occurs currently in the development branch: https://github.com/VDVde/OJP/tree/develop.
+* We work with release-branches
+* PR should always be created with feature/<id> as a branch.
```
Comment (trurlurl) [inline README.md:52]: I don't understand the "feature/<id>" part. I would understand "For every PR a new branch should be created."

File: README.md [hunk 2]
```diff
@@ -32,17 +32,24 @@ at [CEN](https://www.cen.eu).
...
 
+You can contribute by:
+* Pointing out [issues](https://github.com/VDVde/OJP/issues)
+* Creating pull requests: Use the [development documentation](https://github.com/VDVde/OJP/wiki/WG:-:-Working-with-this-repository)
+
 ### Branches
 * The master branch is currently set to OJP 1.0.1
-* Development occurs currently in the changes_for_v1.1 branch.
+* Development occurs currently in the development branch: https://github.com/VDVde/OJP/tree/develop.
+* We work with release-branches
```
Comment (trurlurl) [inline README.md:51]: I don't understand what this means exactly. If it speaks to the more experienced GitHubbers, then that's no problem.

File: README.md (hunk 1 above)
Comment (ue71603) [reply README.md:52]: fixed in the other PR.

File: README.md (hunk 2 above)
Comment (ue71603) [reply README.md:51]: Should be clearer in the other PR now. Will be copied.

**Comment by ue71603** [comment]
@skinkie @trurlurl can you agree?

**Review by trurlurl — APPROVED** [review]

**Comment by ue71603** [comment]
redo based on the latest version

**Pull request closed without merging**